### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by MCWertGaming**
This PR adds issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (Sigmanificient) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: desktop ios?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```
Comment (Sigmanificient) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]:
```suggestion
title: ':bug: '
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```
Comment (Sigmanificient) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]:
```suggestion
title: ':speech_balloon: '
```

**Comment by MCWertGaming** [comment]
Sorry brain dead copy

**Comment by MCWertGaming** [comment]
No, it's right

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (MCWertGaming) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: I mean it's bout the system you are using.

Comment (MCWertGaming) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed it!

**Comment by MCWertGaming** [comment]
Done @Sigmanificient.

**Review by Sigmanificient — APPROVED** [review]

**Pull request merged**